src/components/Player/Pages/Overview/Summary.jsx

```diff
@@ -90,13 +90,15 @@ const SummOfRecMatches = ({ matchesData }) => {
     .toFixed(2));
 
   return (
-    <div className={styles.summaryContainer}>
+    <div>
       <ul>
-        <li>
-          <span>{strings.th_winrate}</span>
-          <p>{winrate}%</p>
-        </li>
-        {Object.keys(computed).map((key) => {
+        {winrate
+          ? <li>
+            <span>{strings.th_winrate}</span>
+            <p>{winrate}%</p>
+          </li>
+          : ''
```

Comment:
Prefer null, empty strings create a stupid <span>. Apparently this will be fixed in react fiber.